build-image-src/build_all_images.sh

```diff
@@ -17,36 +17,39 @@ export DOCKER_CONTENT_TRUST=0
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
 # Single arch images
-docker build -f Dockerfile-java8 -t amazon/aws-sam-cli-build-image-java8:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-nodejs10x -t amazon/aws-sam-cli-build-image-nodejs10.x:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-provided -t amazon/aws-sam-cli-build-image-provided:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-python27 -t amazon/aws-sam-cli-build-image-python2.7:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-python36 -t amazon/aws-sam-cli-build-image-python3.6:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-python37 -t amazon/aws-sam-cli-build-image-python3.7:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-ruby25 -t amazon/aws-sam-cli-build-image-ruby2.5:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
-docker build -f Dockerfile-go1x -t amazon/aws-sam-cli-build-image-go1.x:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION .
+docker build -f Dockerfile-java8 -t amazon/aws-sam-cli-build-image-java8:x86_64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION . &
```

Comment:
This might cause the host to choke, and without pooling it'll get worse every new runtime. It will also make debugging harder as logs will be interleaved.

Are we sure this improves performance significantly?

Reply:
Tested on a CodeBuild with the same compute type as prod. Build time went down from 3 hours to 1.
Although the logs are harder to read, the performance is significant enough.